packages/request-manager/e2e/identities-stress.ts

```diff
@@ -0,0 +1,95 @@
+import path from 'path';
+import fetch, { Response } from 'node-fetch';
+import { createInterceptor, TorController, TorIdentities } from '../src';
+import { torRunner } from './torRunner';
+
+// The purpose of this script is to allow "manual" testing Tor identities changing some parameters.
+// Run it like:
+// yarn workspace @trezor/request-manager test:stress
+// While running this you can run the command below to check that everything is going throw Tor
+// watch -n 1 "lsof -i TCP | grep node"
+
+const host = 'localhost';
+const port = 38835;
+const controlPort = 35527;
+const processId = process.pid;
+const torDataDir = path.join(__dirname, 'tmp');
+const ipRegex = /\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}/;
+
+const INTERCEPTOR = {
+    handler: () => {},
+    getIsTorEnabled: () => true,
+};
+
+const testGetUrlHttps = 'https://check.torproject.org/';
+
+const amountOfIdentities = 20;
+const intervalBetweenRequests = 1000 * 20;
```

Comment:
maybe would be useful to change these in the course of the test - increase amountOfIdentities and decrease intervalBetweenRequests. But on the other hand, this would create a spamming machine :thinking: this is not what we want. How do we stress test without spamming? :thinking: